foxbroadcasting/react-select
# [DPP-856] Make the Drag and Drop components react 15 friendly

## What did you implement:

Implementing issue: [DPP-856](https://fng-jira.fox.com/browse/DPP-856)


## How did you implement it:
Update the drag and drop components to remove calls to `React.PropTypes` and `React.createClass`

## How can we verify it:
1. go to 'fng-cms`
1. Update the `react-select` dependency in`package.json` to be: `"git+ssh://git@github.com:foxbroadcasting/react-select.git#feature/DPP-856"`
1. Run `npm i`
1. Run `npm start`
1. Go to local environment
1. Log in with valid credentials
1. Click Series on sidebar
1. Click New Series
1. Go to Genres, add some genres, the drag and drop should works the same as before


## Screenshots



## Checklist
